### ISSUE TITLE
Tutorial topic: Inheritance and cascading

### ISSUE THREAD

**Issue by trodrigues** [cssconf/CSSclasses--old#42]
The material talks about HTML nesting but not yet about how the CSS cascade works for those cases.

**Comment by bastianalbers** [comment]
Yeah, i think at the end of the tutorial that should be addressed alongside the classes stuff.

**Comment by verpixelt** [comment]
Should be further discussed in the new repository (https://github.com/cssconf/CSSclasses/issues).
